extensions/applicationinsights-properties-js/src/TelemetryContext.ts

```diff
@@ -35,6 +35,17 @@ function _nullResult(): string {
     return null;
 }
 
+function setTageValue(tags: Tags | Tags[], field: string, value: string): void {
+    // Function body
+    if (Array.isArray(tags)) {
```

Comment:
A couple of side notes for minification.

You can use the `isArray` implementation instead of `Array.isArray` (the `isArray` is actually just an alias (constant variable) for the `Array.isArray` (so it's still calling the same thing but it can be minified a LOT smaller.

For the `tags.forEach` we also have an `arrForEach` which also will become smaller and effectively do the same thing (it has a bit more functionality than the `xxx.forEach` but as we use it elsewhere  it's already included so it's "better" (for minification) to use `arrForEach` instead. If this was a performance bottleneck then because of the additional capabilities the `arrForEach` is not as performant (fast) as the `xxx.forEach`